optimism/src/mips/interpreter.rs

```diff
@@ -10,6 +10,10 @@ pub const FD_HINT_WRITE: u32 = 4;
 pub const FD_PREIMAGE_READ: u32 = 5;
 pub const FD_PREIMAGE_WRITE: u32 = 6;
 
+pub const REGISTER_A0: u32 = 4;
+pub const REGISTER_A1: u32 = 5;
+pub const REGISTER_SP: u32 = 29;
```

Comment:
Couldn't find where this constant comes from 😞

Reply:
See https://www.doc.ic.ac.uk/lab/secondyear/spim/node10.html